lib/services/ios-project-service.ts

```diff
@@ -61,6 +62,19 @@ export class IOSProjectService extends projectServiceBaseLib.PlatformProjectServ
 		};
 	}
 
+	public getAppResourcesDestinationDirectoryPath(): IFuture<string> {
+		return (() => {
+			this.$projectDataService.initialize(this.$projectData.projectDir);
+			let frameworkVersion = this.$projectDataService.getValue(this.platformData.frameworkPackageName).wait()["version"];
+
+			if(semver.lt(frameworkVersion, "1.3.0")) {
+				return path.join(this.platformData.projectRoot, this.$projectData.projectName, "Resources", "icons");
+			}
+
+			return path.join(this.platformData.projectRoot, this.$projectData.projectName, "Resources");
```

Comment:
Oh dear, the last part ("icons") went on another line on my monitor and I missed it completely. I am so sorry.